streamz/dataframe/aggregations.py

```diff
@@ -347,8 +348,8 @@ def windowed_groupby_accumulator(acc, new, diff=None, window=None, agg=None, gro
     for o, og in zip(old, old_groupers):
         if 'groupers' in acc:
             assert len(o) == len(og)
-            if hasattr(og, 'index'):
-                assert (o.index == og.index).all()
+#             if hasattr(og, 'index'):
+#                 assert (o.index == og.index).all()
```

Comment:
I would rather not check in commented out code.  Just delete the code and if we need to restore it will be in the file history in Git.

Reply:
Fixed